__tests__/utils-test.js

```diff
@@ -24,3 +28,45 @@ describe('Unit | utils | getNodeProperty', () => {
     expect(getNodeProperty({}, 'volume.level')).toBe(undefined);
   });
 });
+
+describe('Unit | utils | getEmbroiderStrippedPrefixPath', () => {
+  const mockEmbroiderBuildPath =
+    '/private/var/folders/abcdefg1234/T/embroider/098765/tests/acceptance/my-test.js';
+  const mockEmbroiderBuildPathTwoEmbroiderTokens =
+    '/private/var/folders/embroider/abcdefg1234/T/embroider/098765/tests/acceptance/my-test.js';
+  const mockWindowsEmbroiderBuildPath =
+    'C:\\private\\var\\folders\\abcdefg1234\\T\\embroider\\098765\\tests\\acceptance\\my-test.js';
```

Comment:
You don't need to refer to these as mocks. They're just paths that you're passing in.